packages/toolkit/.changes/next-release/Feature-ddc43dca-284c-4ab6-8857-e207bcdd9858.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "provide link to launch ec2 instances from explorer"
```

The changelog is a "mini blog" and is the single most high-traffic way that customers get news about the product. So when writing a changelog, I imagine I'm writing a tweet or a blog post :)

```suggestion
	"description": "EC2: Launch ec2 instances from AWS Explorer"
```